concepticondata/concepticon.tsv

```diff
@@ -1717,7 +1717,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1716	FORTY	Quantity	The natural number forty (40).	Number	
 1717	FIFTY	Quantity	The natural number fifty (50).	Number	
 1718	SIXTY	Quantity	The natural number sixty (60).	Number	
-1719	BURY	The body	to put in the ground and cover with earth.	Action/Process	
+1719	BURY	The body	To put in the ground and cover with earth.	Action/Process	
```

Comment:
Question about this concept in general (I am not sure about the extent of the words already mapped in Concepticon): is it only "bury" as in "bury dead people"? Because if yes, the description could be narrowed down to avoid confusion/uncertainty, "to put a dead body in the ground". If no, and e.g. a buried treasure would fit into this concept as well, "The body" might not be the appropriate semantic category.

Reply:
Based on the mappings (see https://concepticon.clld.org/parameters/1719) it is not specific for "bury people". We could make a "narrower" concept but that should be discussed in an issue first.

Reply:
It's fine if the concept refers to the broader sense. I would just modify the semantic field in that case because "The body" does not really fit if it's not solely about burying corpses. Perhaps "Basic Actions and Technology"?